contrib/libcityhash/include/city.h

```diff
@@ -52,6 +52,10 @@ typedef uint32_t uint32;
 typedef uint64_t uint64;
 typedef std::pair<uint64, uint64> uint128;
 
+/// This is a version of CityHash that predates v1.0.3 algorithm change.
```

Comment:
Thanks! Indeed, this is CityHash v1.0.2. (FYI v1.1 changed the algorithm one more time.)
It seems that `CityHash_v1_0_2` would be more correct than `CityHash64_v1_0_2`.